pkg/pillar/containerd/containerd.go

```diff
@@ -386,8 +394,8 @@ func CtrContainerInfo(name string) (int, string, error) {
 	return 0, "", err
 }
 
-// CtrStartContainer starts the default task in a pre-existing container and attaches its logging to memlogd
-func CtrStartContainer(domainName string) (int, error) {
+// CtrCreateTask starts the default task in a pre-existing container and attaches its logging to memlogd
+func CtrCreateTask(domainName string) (int, error) {
```

Comment:
Was this actually used before? For a while, we were starting containers, but then we switched to creating a snapshot and letting Xen or kvm create a new image based on the mounted directory via p9.

Reply:
Not sure I understand the question: what do you mean 'before' @deitch ?